Kickstarter-iOS/Views/Controllers/RewardsCollectionViewControllerTests.swift

```diff
@@ -31,7 +31,7 @@ final class RewardsCollectionViewControllerTests: TestCase {
         parent.view.frame.size.height = device == .pad ? 1_800 : 1_000
         parent.view.frame.size.width = device == .pad ? 2_300 : 1_800
 
-        FBSnapshotVerifyView(vc.view, identifier: "lang_\(language)_device_\(device)", tolerance: 0.01)
+        FBSnapshotVerifyView(vc.view, identifier: "lang_\(language)_device_\(device)")
```

Comment:
@ifbarrera do you remember what was the problem originally when we decided to use `tolerance`?

For some reason using `tolerance` (and no matter what value) would pass this test even with the checkmark on so clearly that wasn't ideal.

I've re-recorded snapshots with checkmark hidden (seems like no difference at all)..and after removing tolerance tests pass for checkmark hidden, fail for checkmark shown - as it should (we don't want to show it for now - there's no business logic behind it).

Reply:
I can't remember exactly. Feel free to remove!